src/app/components/forms/game/sketchfab/sketchfab.ts

```diff
@@ -1,17 +1,56 @@
-import { BaseForm, FormOnInit } from '../../../../../_common/form-vue/form.service';
+import { Component, Prop } from 'vue-property-decorator';
+import { BaseForm, FormOnInit, FormOnSubmit } from '../../../../../_common/form-vue/form.service';
 import { Game } from '../../../../../_common/game/game.model';
 import { GameSketchfab } from '../../../../../_common/game/sketchfab/sketchfab.model';
-import { Component, Prop } from 'vue-property-decorator';
+import AppSketchfabEmbed from '../../../../../_common/sketchfab/embed/embed.vue';
 
-@Component({})
-export default class FormGameSketchfab extends BaseForm<GameSketchfab> implements FormOnInit {
+@Component({
+	components: {
+		AppSketchfabEmbed,
+	},
+})
+export default class FormGameSketchfab extends BaseForm<GameSketchfab>
+	implements FormOnInit, FormOnSubmit {
 	@Prop(Game) game!: Game;
 
+	readonly SKETCHFAB_URL_REGEX_1 = /^(?:(?:https:\/\/)?(?:www.)?(?:sketchfab.com\/3d-models\/(?:[a-z0-9]+-)+([0-9a-f]{32})\/?))$/i;
+	readonly SKETCHFAB_URL_REGEX_2 = /^(?:(?:https:\/\/)?(?:www.)?(?:sketchfab.com\/models\/([0-9a-f]{32})\/?))$/i;
+	readonly SKETCHFAB_FIELD_REGEX = /^(?:(?:https:\/\/)?(?:www.)?(?:sketchfab.com\/3d-models\/(?:[a-z0-9]+-)+([0-9a-f]{32})\/?))$|^(?:(?:https:\/\/)?(?:www.)?(?:sketchfab.com\/models\/([0-9a-f]{32})\/?))$|^([0-9a-f]{32})$/i;
```

Comment:
This is kind of unreadble now. Also, whats the difference between url regex and field regex? can you put a little comment explaining the use case for these?